### ISSUE TITLE
enhancement : Change the name of 'localhost:1001/administrator' site to some thing creative

### ISSUE THREAD

**Issue by dineshkummarc** [ProcessManagement/Erls-Gitlab-Demo#1]
Bug Summary: Change the name of 'localhost:1001/administrator' site to some thing creative
Environment : Firefox37.0.2
Regression:
precondit ion:
St eps t o reproduce:
Expect ed result : change the name of 'localhost:1001/administrator' site to some thing creative
Act ual result : name of 'localhost:1001/administrator' site is gitlab, which resembles with gitlab.com website.
Comment s:
1. Please find attached screenshots " " for more detail info.

**Comment by dineshkummarc** [comment]
what about the name 'erlslab'

**Comment by dineshkummarc** [comment]
and can be hosted on domain erlslab.erlsindia.co.in or erlslab.erlsindia.com

**Comment by dineshkummarc** [comment]
Added discussion and suggestion labels for
discussing further on this topic.

**Comment by dineshkummarc** [comment]
the new name suggested by partner seems to be like ErlsHUB. what about this
new name. suggest me.

**Comment by ghost** [comment]
Yes ErlsHub can be finalised. and can be hosted on domain ErlsHub.erlsindia.co.in or ErlsHub.erlsindia.com

**Comment by dineshkummarc** [comment]
ok thanks for the confirmation.
